recipe/meta.yaml

```
@@ -9,12 +8,12 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 93e60a40988f4d65e9f4885df477e2fdaff6b73a9ded434c1ab356dd57eefff7
 
 build:
   number: 0
-  skip: true  # [py<37]
-  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [py<38]
+  script: python -m pip install --no-deps --ignore-installed . --no-build-isolation
```

```suggestion
  script: python -m pip install . --no-deps --ignore-installed --no-build-isolation
```
Moving a dot `.` for readability